instance/tests/models/test_openedx_appserver.py

```diff
@@ -151,6 +151,11 @@ def test_provision_failed(self, mocks, mock_consul):
         self.assertFalse(result)
         self.assertEqual(appserver.status, AppServerStatus.ConfigurationFailed)
         self.assertEqual(appserver.server.status, Server.Status.Ready)
+
+        # after configuration fail we run additional playbook to stop services
+        self.assertEqual(mocks.mock_run_ansible_playbooks.call_count, 1)
```

Comment:
This assertion is a bit broad IMO, I think mocking `manage_instance_services` and then `mock_manage_instance_services.assert_called_once_with(active=False)` makes the intent clearer.